raiden/storage/sqlite.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
+import sqlite3
+import threading
+
+
+class SQLiteStorage:
```

Comment:
So when this class is integrated we can get rid of `StateChangeLogSQLiteBackend` in transfer/log.py?

Reply:
yes